shell/TestPackage.ps1

```diff
@@ -0,0 +1,24 @@
+$ErrorActionPreference = "Stop"
+$LastExitCode = 0
+$env:PATH +=";$($env:SystemDrive)\\ProgramData\\chocolatey\\bin"
+
+# https://github.com/chocolatey/choco/issues/512
+$validExitCodes = @(0, 1605, 1614, 1641, 3010)
+
+$package  = ls c:\packages\*.nupkg
+if (!$package) { $package = ls c:\packages\remote\*.nupkg; $remote=$true }
+if (!$package) { Write-Host 'No packages found in directory packages or packages\remote'; return }
```

Comment:
How would you handle the difference ? How would you mark whats needed for remote and what with local source ?